### PR TITLE
Improve Arabic support

### PR DESCRIPTION
**Pull request by richtier**
[this task](https://uktrade.atlassian.net/browse/ED-1723)

### CODE REVIEW

File: enrolment/templates/marketing-pages/bidi/base.html [hunk 1]
```diff
@@ -2,7 +2,6 @@
 {% load static from staticfiles %}
 {% load trans from i18n %}
 {% block css %}
-	<link href="{% static 'marketing-page.css' %}" media="all" rel="stylesheet" />
 	<link href="{% static 'marketing-page-bidi.css' %}" media="all" rel="stylesheet" />
```
Comment (richtier) [inline enrolment/templates/marketing-pages/bidi/base.html]: `marketing-page-bidi.css` now contains all the css that was in `marketing-page.css`, so no need to download both files.

**Comment by richtier** [comment]
@stevenburnell-dit you're correct about the `.css` typos. They should be `.scss`.

I'm calling `@main` multiplte times instead of doing this in the html file:

```
	<link href="{% static 'landing-page.css' %}" media="all" rel="stylesheet" />
  	<link href="{% static 'landing-page-bidi.css' %}" media="all" rel="stylesheet" />
```

The outcome is the browser recieves the same css, but by requesting only one css file instead of two. Slightly quicker.

**Comment by richtier** [comment]
see [this branch](https://github.com/uktrade/directory-ui-supplier/pull/232)

**Comment by stevenburnell-dit** [comment]
I’m not sure that’s a solid plan and would expect it to be slower rather than quicker. 

I’d give three reasons:

1. 
Considering the ability of HTTP persistent connections, I’d argue developers no longer need to consolidate multiple requests into a single, large, effort. I believe HTTP 1.1 has it switched on by default and, with HTTP 2.0, it adds to that by supporting multiple concurrent requests and responses over a single connection. 

All this meaning that we’re no longer penalised by such things as the additional connection times of multiple, individual requests, and the delay is now more about actual download time. 

Given that, if you’re calling @main multiple times, that leads me to believe we’re bloating the CSS, by simple code duplication. In effect, you’ll be decreasing speed even with a persistent connection because you’re simply increasing the download size. 

2.
It all depends on the importance of priority order but, it should be safe to assume that, by calling the Main CSS multiple times, you’re likely running a very high risk of breaking the specificity of CSS rules with unpredictable and unexpected results. The resulting effect is likely going to get quite messy with your last stylesheet adding rules to overwrite the winning rule, that you didn’t want, that is overwriting the one that you did, that previously overwrote the one you’re now having to compensate for, once again. Yes, intentionally sounding confusing because that is very likely to be what happens when you call the same file multiple times. 

3. 
Another often overlooked impact is that, by adding to the size of CSS, we’re actually forcing the browser to do a hell of a lot more parsing and processing than it would if we only requested the CSS we actually expect to be used on the page we’re looking at. This is a problem for all those sites that choose to follow the web-app methodology of preloading the entire set of CSS files ever required. In a single-page (app) environment you’re going to suffer that hit once, so you might consider it worth overlooking but, in a multiple-page (site) environment, you’re going to hit that overhead on each page.   By including the Main CSS multiple times, we might be sticking only to CSS that is required for the page, but we’re still causing more effort by asking the browser to parse the same chunk of code more than once. 


Have I misunderstood what you’re telling me?


BTW. I looked at the headers for FAS and noticed that we don’t appear to have persistent connections enabled for AWS. I’m currently assuming that’s probably due to the mis-configuration you’ve recently fixed, but it might be worth checking out, if not.

**Pull request closed without merging**